ui/components/component-library/button-icon/button-icon.stories.js

```diff
@@ -94,16 +94,16 @@ export default {
 export const DefaultStory = (args) => <ButtonIcon {...args} />;
 
 DefaultStory.args = {
-  iconName: ICON_NAMES.CLOSE,
+  iconName: IconName.Close,
   ariaLabel: 'Close',
 };
 
 DefaultStory.storyName = 'Default';
 
-export const IconName = (args) => <ButtonIcon {...args} />;
-
-IconName.args = {
-  iconName: ICON_NAMES.CLOSE,
+export const IconNameStory = (args) => <ButtonIcon {...args} />;
+IconNameStory.storyName = 'IconName';
+IconNameStory.args = {
+  iconName: IconName.Close,
```

Comment:
## After
ButtonIcon story still works and shows up in docs


https://user-images.githubusercontent.com/8112138/227678044-75ed7416-ede1-4b9d-b967-1cda4f004c0b.mov